qiskit/visualization/qcstyle.py

```diff
@@ -56,23 +56,23 @@ def __init__(self):
             'z': 'Z',
             'h': 'H',
             's': 'S',
-            'sdg': 'S^\\dagger',
+            'sdg': '$S^\\dagger$',
```

Comment:
I think it would be nice to have the standard gate names either all be slanted (i.e. math mode) or not. Could we therefore also have the others wrapped in `$$`? Or alternatively, to have them non-slanted, we could do `$\mathrm{S}^\dagger$`.

Reply:
I don't have a strong opinion on this, I would lean towards making everything not slanted so that all gate names/labels aren't slanted. Unless you think having standard gates slanted is a good way to differentiate them from other gates?

Reply:
The gates in the IQX composer will be non-slanted so we should make them all non-slanted here as well. 🙂

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/4669/commits/6955736918a0eedd53cecf8d5bcbe38162ba95f0